apps/rackstep.ru

```diff
@@ -0,0 +1,13 @@
+require 'rackstep'
+
+class HelloWorld < RackStep::App
+  add_route('GET', '', 'HelloWorldController')
```

Comment:
Not sure if this will only answer to root (`/`) or will answer to any path given (`/foo`). Can you please confirm that?

The only path that needs to answer is the root one.

Thank you.